### PR TITLE
update ledger welcome page and partner logos

### LINKED ISSUE #3353: update ledger welcome page with adjusted UI layout and partner logos
image

showing some dims:
image

partner logos attached

[partner logos.zip](https://github.com/brave/browser-laptop/files/433649/partner.logos.zip)

### PR DESCRIPTION
**Pull request by jkup**
- [x] Submitted a [ticket](https://github.com/brave/browser-laptop/issues) for my issue if one did not already exist.
- [x] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [x] Ran `git rebase -i` to squash commits if needed.

Fix #3353 

Tagging @bbondy @bradleyrichter 

I guessed on a few of these font sizes, let me know if you'd like them different! 

A few things I wanted to run by you all:

- I would have rather used img tags for the logos, but I saw the only other place we have retina images we used this image-set approach. Should I do it this way? With empty divs?
- I changed a lot in the translation files. Just wanted to make sure making those kind of changes isn't going to break anything for translations!

### CODE REVIEW

**Comment by bsclifton** [comment]
Just checking it out now... the images aren't showing for me :confused: 
screen shot 2016-08-24 at 11 24 20 am

Console (cmd+option+i) has:
```
chrome-extension://mnojpmjdmbbfmejpflffifhffcmidifd/gen/4bb98a05900622845d2e66b433711e32.png Failed to load resource: net::ERR_FILE_NOT_FOUND
chrome-extension://mnojpmjdmbbfmejpflffifhffcmidifd/gen/1b0aa5afd818eff2fcc8b4386a8bc38b.png Failed to load resource: net::ERR_FILE_NOT_FOUND
chrome-extension://mnojpmjdmbbfmejpflffifhffcmidifd/gen/3caa6d84e23f04adf06c13d4063cda48.png Failed to load resource: net::ERR_FILE_NOT_FOUND
```

**Comment by jkup** [comment]
@bsclifton :( that stinks. I have no idea. Do you see those images on your filesystem? They should be in: browser-laptop/app/extensions/brave/img/

**Comment by cezaraugusto** [comment]
@jkup @bsclifton I faced the same issue while linking images on new tab page. Looks like there's a delay (likely from Electron?) that makes new files unavailable for a while. Crashing and then reopening the page did the trick here.

You can also try double-checking with `npm run watch-all` to see if the file was emitted.

**Comment by bbondy** [comment]
@bsclifton pls review
@jkup this needs a rebase, thx.

```
git pull --rebase
git push -f
```

...will update the PR in place.

File: app/extensions/brave/locales/en-US/preferences.properties [hunk 1]
```diff
@@ -13,12 +13,18 @@ publishers=Publishers
...
+paymentsWelcomeText2=To begin using Brave Payments, simply flip the switch at the top of this window. The rest is easy.
+paymentsWelcomeText3=Brave Payments allows publishers to stay in business even though you may have blocked their advertisements with Brave. All of this works while keeping your browsing history private. Your funds are transferred to the site owner through an anonymous ledger sytem, which makes it impossible for you ever to be identified based on the sites you visit and support.
+paymentsWelcomeText4=Brave Payments is currently in Beta. With your help and feedback, we can fine tune the system through the beta period.
+paymentsWelcomeText5=Need more info?
+paymentsWelcomeText6=for Brave Payments...
+paymentsWelcomeLink=View the FAQ
+publisherPaymentsTitle=Brave Payments
+publisherPaymentsTitleBeta=beta
+paymentsSidebarText1=Our Partners
+paymentsSidebarText2=All transactions are anonymized with technology from:
```
Comment (diracdeltas) [inline app/extensions/brave/locales/en-US/preferences.properties]: @bradleyrichter seems like we should be more clear here. `anonymize` can mean a lot of things. all that we use PIA for is hiding real IP addresses. they do not anonymize payments, etc.

Comment (diracdeltas) [inline app/extensions/brave/locales/en-US/preferences.properties]: is `All transaction IP addresses are anonymized with technology from:` too long?

File: js/about/preferences.js [hunk 2]
```diff
@@ -623,13 +630,17 @@ class PaymentsTab extends ImmutableComponent {
...
             <div data-l10n-id='paymentsWelcomeText1' />
-            <div data-l10n-id='paymentsWelcomeText2' />
+            <div className='boldText' data-l10n-id='paymentsWelcomeText2' />
             <div data-l10n-id='paymentsWelcomeText3' />
-            <div className='boldText' data-l10n-id='paymentsWelcomeTextBold' />
-            <a href='https://github.com/brave/ledger/blob/master/documentation/Ledger-Principles.md' target='_blank' data-l10n-id='paymentsWelcomeText4' />
+            <div data-l10n-id='paymentsWelcomeText4' />
+            <div>
+              <span data-l10n-id='paymentsWelcomeText5' />&nbsp;
+              <a href='https://github.com/brave/ledger/blob/master/documentation/Ledger-Principles.md' target='_blank' data-l10n-id='paymentsWelcomeLink' />&nbsp;
```
Comment (diracdeltas) [inline js/about/preferences.js]: this needs to be replaced with the actual FAQ link, cc @bradleyrichter

**Comment by bsclifton** [comment]
Weird- the images are still not showing for me.  I used dev tools to disable cache and also tried crashing page, per suggestion of @cezaraugusto

**Comment by bsclifton** [comment]
Update- works great for me (and looks good, @jkup!) :smile: ++

I hadn't done an npm install in a while. Something during that process fixed it up

File: app/extensions/brave/locales/en-US/preferences.properties (hunk 1 above)
Comment (bradleyrichter) [inline app/extensions/brave/locales/en-US/preferences.properties]: this fits. thanks.

**Comment by bradleyrichter** [comment]
new URL for brave payments FAQ page:

https://brave.com/Payments_FAQ.html

**Comment by jkup** [comment]
AFAIK this is good to go. Does it look good now @bradleyrichter @diracdeltas

**Comment by bbondy** [comment]
@diracdeltas please review when you can, I'm going to merge in though for beta5.

**Comment by diracdeltas** [comment]
are the logos supposed to go to the bottom like this when the window is narrow? if so, lgtm
screen shot 2016-08-27 at 12 16 45 am

**Comment by bradleyrichter** [comment]
Ideally the text area should shrink to make room for the logos down to a min acceptable width. (400px?) Then everything stays. 

If we don't have time for this, let's just have the logos stay put.

**Comment by incitadus** [comment]
I've been using Brave for about four days now and now receiving this error... brave won't
    load....going to reinstall...... I really like this browser.
    
    Your file was not found

    It may have been moved or deleted.
    ERR_FILE_NOT_FOUND

**Comment by bsclifton** [comment]
@incitadus do you have more info? I'm trying to understand what the problem you're having is

When you open Brave, is that the error you're getting?  Or is the error related to Brave Payments?

What operating system are you running? (Windows 10?).  Please let me know and I can help 😄 👍

**Comment by incitadus** [comment]
That's the error I get when trying to open Brave. Using win10.  Worked for
about a week
then the error started when I pinned to my start menu.Have deleted and
reinstalled several x's w/o
success. Thanks for your help.

**Pull request merged**